test/Tests/Adapters/Engine/AbstractTypeConstructionTests.cs

```diff
@@ -31,7 +31,7 @@ public async Task Can_Execute_Query_On_Schema_From_Abstract_Types()
                 .Execute();
 
             result.ShouldHaveNoErrors();
-            result.Data.ShouldHaveFieldWithValue("commonField", new DateTime(1970, 1, 1));
+            result.Data.ShouldHaveFieldWithValue("commonField", "1970-01-01");
```

Comment:
Having to change a unit-test like that should ring a bell. The fact that the `CommonField` object is no longer a `DateTime` but a `string` in the newer version sounds like a bug that needs to be fixed.

Reply:
I tried to step through the code for the different versions, but it's unclear where things go wrong :)

Reply:
The failing test here is caused by the introduction of new DateTimeGraphType and DateTimeOffsetGraphType. I've created an other pull-request that includes these changes and updates to version 2.1.0, so this PR can be closed.

See #106

Reply:
the change in `graphql-dotnet`: https://github.com/graphql-dotnet/graphql-dotnet/pull/665/files#diff-6ecb3c17fef816d6cd02a67ad408ea9eR19